independent-projects/qute/core/src/main/java/io/quarkus/qute/Locate.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.qute;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import io.quarkus.qute.Locate.Locates;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * A custom {@link TemplateLocator}s are not available during the build time, therefore {@link Template} located by
+ * the locator must disable its validation by annotating the {@link TemplateLocator} with this annotation. If
+ * {@link TemplateLocator} locates {@link Template} and fails to declare the fact this way, an exception is thrown and the build
+ * fails.
+ *
+ * Quarkus automatically register {@link TemplateLocator}s annotated wth '@Locate'.
```

Comment:
This is not entirely true - in fact, all CDI beans that implement `TemplateLocator` are registered... which is fine IMO!